### PR TITLE
mesos-dns: set listeners to localhost

### PR DESCRIPTION
**Pull request by alberts**
@sargun @sargun @adam-mesos leftover from one of our internal branches... thoughts?

### CODE REVIEW

**Comment by alberts** [comment]
We already have this change for our DC/OS distribution:

https://github.com/mesosphere/dcos-enterprise/blob/master/gen_extra/dcos-config.yaml#L69

**Comment by spahl** [comment]
Even the normal listener should be 127.0.0.1. spartan is the dns front

**Comment by cmaloney** [comment]
This is reliably failing tests, moving to work in progress.

File: gen/dcos-config.yaml [hunk 1]
```diff
@@ -71,7 +71,8 @@ package:
         "port": 61053,
         {{ mesos_dns_resolvers_str }},
         "timeout": 5,
-        "listener": "0.0.0.0",
+        "listener": "127.0.0.1",
```
Comment (sargun) [inline gen/dcos-config.yaml]: If you do this, how will the agents ever query Mesos-DNS?

**Comment by sargun** [comment]
Spartan doesn't front Mesos-DNS requests from the agents. We can make it do that, but I think last time we had this discussion we came away against that for reliability reasons.

**Pull request closed without merging**